graph/breadth_first_search.cpp

```diff
@@ -77,8 +77,8 @@ void addEdge(std::vector<std::vector<int>> &adj, size_t u, size_t v) {
  * @param start vertex from where traversing starts
  *
  */
-void beadth_first_search(std::vector<std::vector<int>> &adj, size_t start) {
-    size_t vertices = adj.size();
+void beadth_first_search(std::vector<std::vector<int>> *adj, size_t start) {
+    size_t vertices = (*adj).size();
```

Comment:
for pointers, we use 
```suggestion
    size_t vertices = adj->size();
```